tartufo/commands/scan_local_repo.py

```diff
@@ -31,7 +31,7 @@ def main(
 ) -> Tuple[str, List[Issue]]:
     """Scan a repository already cloned to your local system."""
     git_options = types.GitOptions(
-        since_commit=since_commit, max_depth=max_depth, branch=branch
+        since_commit=since_commit, max_depth=max_depth, branch=branch, local_only=True
```

Comment:
We should expose this as an option to the user so that we can maintain the same default behavior as discussed in the other comments. With all the other work you've done it should just be a matter of adding another `@click.option('--fetch/--no-fetch', ...)` similar to what can be seen in `cli.py`. Then pass that bool in to this enum. (Or the negation of that bool rather, since `--no-fetch` would mean `local_only=True`. Unless you feel like all the effort of switching to using a verb-style option instead of descriptive. That's not necessary though.)